src/Concerns/ManagesFileResponses.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace LaraCrafts\ChunkUploader\Concerns;
+
+trait ManagesFileResponses
```

Comment:
This response is not needed anymore. An event is triggered when the upload finished. Later on a progress event can be created as well.